flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -649,6 +649,13 @@ public boolean dropTemporaryTable(String path) {
         }
     }
 
+    @Override
+    public boolean dropTable(String path) {
+        UnresolvedIdentifier unresolvedIdentifier = getParser().parseIdentifier(path);
+        ObjectIdentifier identifier = catalogManager.qualifyIdentifier(unresolvedIdentifier);
+        return catalogManager.dropTable(identifier, true);
```

Comment:
just to clarify: this is different from SQL right? `DROP TABLE` in SQL will use `ifNotExists=false`. If yes we should also document this in the JavaDoc to avoid confusion.

```
Compared to SQL, this method will not throw an error if the table does not exist. Use {@link #dropTable(.., ignoreIfExists)} to change the default behavior.
```

Reply:
Same for views

Reply:
shouldn't `true` be `ignoreIfNotExists` here

Reply:
i didn't get it , could you elaborate?

Reply:
sure. the code is `return catalogManager.dropTable(identifier, true);` but should be 
`return catalogManager.dropTable(identifier, ignoreIfNotExists);` otherwise it does not pass the flag through

Reply:
got it, thanks
fixed, also added a test for that case